storage/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteLogManagerConfig.java

```diff
@@ -370,6 +384,10 @@ public long remoteLogIndexFileCacheTotalSizeBytes() {
         return remoteLogIndexFileCacheTotalSizeBytes;
     }
 
+    public long getRemoteLogIndexFileCacheTotalSize() {
```

Comment:
In Kafka code base we don't use prefix such as "get" for getters. You can rename this to `remoteLogIndexFileCacheTotalSize()`